src/components/visual/explorer-link/explorer-link.tsx

```diff
@@ -19,19 +21,30 @@ export class ExplorerLink {
   @State() hasSlotContent: boolean = false;
 
   componentWillLoad() {
-    this.hasSlotContent = !!this.hostElement.querySelector('[slot="content"]');
+    this.hasSlotContent = !!this.hostElement.querySelector(`[slot="${EXPLORER_LINK_SLOT_NAME}"]`);
   }
 
   render() {
     let icon = 'faArrowUpRightFromSquare';
 
     if (this.icon) {
-      icon = typeof this.icon === 'string' ? getIconHtmlFromIconName(this.icon) : getIconHtmlFromIconDefinition(this.icon);
+      icon =
+        typeof this.icon === 'string' ? getIconHtmlFromIconName(this.icon) : getIconHtmlFromIconDefinition(this.icon);
     }
 
     return (
-      <a data-testid={this.dataTestId} href={this.link} target="_blank" class={{ 'mvx:explorer-link': true, [this.class]: Boolean(this.class) }} rel="noreferrer">
-        {this.hasSlotContent ? <slot name="content"></slot> : this.text ?? <mvx-fa-icon icon={icon} class={this.iconClass}></mvx-fa-icon>}
+      <a
+        data-testid={this.dataTestId}
+        href={this.link}
+        target="_blank"
+        class={{ 'explorer-link': true, [this.class]: Boolean(this.class) }}
+        rel="noreferrer"
+      >
+        {this.hasSlotContent ? (
+          <slot name={EXPLORER_LINK_SLOT_NAME} />
```

Comment:
why still use text?

Reply:
Will be solved in future `Pull Request`. 👍